pkg/util/fs_utils.go

```diff
@@ -160,3 +160,22 @@ func CheckSameFile(f1name, f2name string) (bool, error) {
 	}
 	return true, nil
 }
+
+// HasFilepathPrefix checks if the given file path begins with prefix
+func HasFilepathPrefix(path, prefix string) bool {
+	path = filepath.Clean(path)
+	prefix = filepath.Clean(prefix)
+	pathArray := strings.Split(path, "/")
```

Comment:
Can you use filepath.Split here?

Reply:
I think it only [splits](https://golang.org/pkg/path/filepath/#Split) along the last separator

Reply:
D'oh. Right.